movedata/move.go

```diff
@@ -71,31 +94,32 @@ func (mover *Mover) MoveData() error {
 	if err != nil {
 		return err
 	}
+
 	for rows.Next() {
 		columns := make([]interface{}, len(cols))
-		columnPointers := make([]interface{}, len(cols))
 		for i := range columns {
-			columnPointers[i] = &columns[i]
+			columns[i] = &columns[i]
```

Comment:
columns is not initialized, what you want to do with this cycle:
```golang
	for i := range columns {
			columns[i] = &columns[i]
		}
```
columns is filled here later:
`rows.Scan(columns...)`

Reply:
Here we need to understand how to define an array of pointers, I agree that this "initialization" cycle is ugly and useless, but it's the only way I actually found to cast the array elements to pointer.